src/telemetry/telemetryService.ts

```diff
@@ -62,7 +62,11 @@ export class TelemetryService {
   }
 
   public getCommonProperties(): Record<string, string | undefined> {
-    return { extension_version: `${version}`, device_id: this.deviceId };
+    return {
+      extension_version: `${version}`,
+      device_id: this.deviceId,
+      app_name: vscode.env.appName, // e.g., "Visual Studio Code" or "Azure Data Studio"
```

Comment:
could this be undefined? we should fallback to `Visual Studio Code - Unknown` or something like that

Reply:
yeah, just makes it more intentional / easier for us to track what scenario happened